docs/conf/portland/2022/cfp_email_templates.rst

```diff
@@ -9,24 +9,99 @@ Speaker email templates
 {% if cfp_variables['print_templates'] and not flagpostconf %}
 
 .. contents::
-    :local:
-    :depth: 1
-    :backlinks: none
+   :local:
+   :depth: 1
+   :backlinks: none
 
 CFP  - Acceptance template
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-TBD
+Tickets:
+   {{cfp_variables['ticket']}}
+
+--------------------------------------
+
+Subject:
+   WTD {{city}} {{year}} -- talk accepted!
+
+::
+
+   Hi {name},
+
+   The Write the Docs talk selection committee has just wrapped up the review process and we'd love it if you could join us as a speaker!
+
+   Write the Docs {city} is held in **{city} on {date}**.
+
+   We think your '{title}' talk would be a great fit for the conference. We'd love to have you prepare it for a **30-minute** time slot.
+
+   So, from here, your first step is to reply to this email as soon as possible, to confirm both of the following:
+
+   * You're still interested in presenting in {city} on {date}. We'll confirm the specific day and time for your talk at a later date.
```

Comment:
```suggestion
   * You're still interested in presenting for {city} on {date}. We'll confirm the specific day and time for your talk at a later date.
```

we might need a different template for virtual confs — but changing this preposition is a smol workaround